rm-datasource/src/main/java/io/seata/rm/datasource/exec/InsertExecutor.java

```diff
@@ -306,6 +306,11 @@ private boolean checkPkValues(List<Object> pkValues) {
             Object v = genKeys.getObject(1);
             pkValues.add(v);
         }
+        try {
+            genKeys.beforeFirst();
+        } catch (SQLException e) {
+            LOGGER.warn("Fail to reset ResultSet cursor. can not get primary key value");
```

Comment:
This execption should be ignore?

Reply:
only user not need get primary key, can be ignore exception.